### ISSUE TITLE
Class and Profession trainers not working for NEW lvl 70 druid

### ISSUE THREAD

**Issue by Bambbi** [dalaranwow/dalaran-wow#4875]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen, I should have access to trainers 
**Steps to reproduce the problem**:

1. Pay cash for donate points
2. Use donate point to raise existing druid to lvl 70
3. HAVE NO ACCESS TO TRAINERS OF ANY KIND

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achieveme
![wowscrnshot_091317_201406](https://user-images.githubusercontent.com/31943537/30409120-476527a8-98c0-11e7-91ed-efea79d15843.jpg)

![wowscrnshot_091317_201209](https://user-images.githubusercontent.com/31943537/30409115-42382942-98c0-11e7-8569-6805a0e60e52.jpg)
nts, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
